modules/guides/pages/create-user-defined-function.adoc

```diff
@@ -80,6 +80,6 @@ include::example$javascript-udfs/create-scoped-n1ql-udf.n1ql[]
 --
 ====
 
-NOTE: The N1QL User-Defined Function will take the same scope as the JavaScript UDF it is referencing.
+NOTE: The {sqlpp} User-Defined Function will take the same scope as the JavaScript UDF it is referencing.
 
-Having created your N1QL User-Defined Function, the xref:call-user-defined-function.adoc[next step] is to use  N1QL statement to call the function.
+Having created your {sqlpp} User-Defined Function, the xref:call-user-defined-function.adoc[next step] is to use an  {sqlpp} statement to call the function.
```

Comment:
As above, I think we should leave the indefinite article unchanged; it's one less thing to worry about.